dbos_transact/context.py

```diff
@@ -290,8 +290,9 @@ def __exit__(
         exc_value: Optional[BaseException],
         traceback: Optional[TracebackType],
     ) -> Literal[False]:
-        assert assert_current_dbos_context().is_within_workflow()
-        assert_current_dbos_context().end_workflow(exc_value)
+        ctx = assert_current_dbos_context()
+        assert ctx.is_within_workflow()
+        ctx.end_workflow(exc_value)
```

Comment:
Just curious. What was the bug here? `assert_current_dbos_context()` didn't return the correct one?

Reply:
I misread it in the merge.